docs/workflow/trimming/ILLink-files.md

```diff
@@ -0,0 +1,45 @@
+# ILLink Files
+
```

Comment:
This is great - thanks a lot!
I would just mention that the format of these files are described in https://github.com/mono/linker/blob/main/docs/data-formats.md.

Reply:
Is line 5 enough?

> See https://github.com/mono/linker/blob/main/docs/data-formats.md for full documentation on these files.

Reply:
Sorry - I completely missed that one - please ignore...